components-rails.gemspec

```diff
@@ -6,12 +6,11 @@ require "components/rails/version"
 Gem::Specification.new do |spec|
   spec.name          = "components-rails"
   spec.version       = Components::Rails::VERSION
-  spec.authors       = ["Ben Hutton"]
-  spec.email         = ["benhutton@gmail.com"]
+  spec.authors       = ["Joe Osburn", "Ben Hutton", "Desiring God"]
+  spec.email         = ["joe@jnodev.com", "benhutton@gmail.com", "web@desiringgod.org"]
 
-  spec.summary       = %q{TODO: Write a short summary, because Rubygems requires one.}
-  spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+  spec.summary       = %q{Add Component support to Rails.}
```

Comment:
@benhutton Hmm, sorry, my comment didn't show up. Might have been related to the outage, or possibly user error (on my part.)

Could we change this to say "web components" or possibly "front end components" or something like that?